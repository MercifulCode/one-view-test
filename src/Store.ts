import { createStore, applyMiddleware } from "redux";
import thunk from "redux-thunk";
import { jsonReducer } from "./reducers/JSONReducer";

export const store = createStore(jsonReducer, applyMiddleware(thunk));
