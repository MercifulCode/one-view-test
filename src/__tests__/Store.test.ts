import { store } from "../Store";

describe("App Store", () => {
  it("Should match the existing snapshot.", () => {
    expect(store).toMatchSnapshot();
  });
});
