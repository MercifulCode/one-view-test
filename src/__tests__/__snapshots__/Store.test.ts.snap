// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Store Should match the existing snapshot. 1`] = `
Object {
  "dispatch": [Function],
  "getState": [Function],
  "replaceReducer": [Function],
  "subscribe": [Function],
  Symbol(observable): [Function],
}
`;
